src/pages/es/guides/server-side-rendering.mdx

```diff
@@ -102,11 +102,11 @@ También puedes añadir un adaptador manualmente instalando el paquete y actuali
 
 Astro seguirá siendo un generador de sitios estáticos por defecto. Pero una vez que habilites un adaptador de renderizado en el servidor, **cada ruta en la carpeta de páginas se convertirá en una ruta renderizada por el servidor** y algunas características nuevas estarán disponibles para ti.
 
-### Prerenderizado
+### Renderizado Híbrido
 
-Since v="2.0.0-beta.0" />
+Since v="2.0.0" />
 
-Cualquier archivo `.astro` dentro de `pages/` puede optar por el prerenderizado, o el comportamiento estático de construcción estándar, incluyendo la siguiente línea.
+Para habilitar **renderizado híbrido**, cualquier [página](/es/core-concepts/astro-pages/) o [endpoint del servidor](/es/core-concepts/endpoints/#endpoints-del-servidor-rutas-de-api) puedes optar por el prerenderizado. Estos archivos se renderizarán estáticamente en tiempo de compilación, similar al modo estático por defecto [`output`](/es/reference/configuration-reference/#output).
```

Comment:
```suggestion
Para habilitar **renderizado híbrido**, cualquier [página](/es/core-concepts/astro-pages/) o [endpoint del servidor](/es/core-concepts/endpoints/#endpoints-del-servidor-rutas-de-api) puede optar por un renderizado híbrido. Estos archivos se renderizarán estáticamente en tiempo de compilación, similar al modo estático por defecto [`output`](/es/reference/configuration-reference/#output).
```

Reply:
El commit dice esto: 
"To enable **Hybrid Rendering**, any [page](/en/core-concepts/astro-pages/) or [server endpoint](/en/core-concepts/endpoints/#server-endpoints-api-routes) `can opt-in to prerendering.`
These files will be statically rendered at build-time, similar to the default static [`output`](/en/reference/configuration-reference/#output) mode."

Reply:
Si creo que lo dejamos en prerenderizado